pkg/services/searchV2/index.go

```diff
@@ -308,6 +315,11 @@ func (l sqlDashboardLoader) LoadDashboards(ctx context.Context, orgID int64, das
 		}
 
 		for _, row := range rows {
+			info, err := extract.ReadDashboard(bytes.NewReader(row.Data), lookup)
+			if err != nil {
+				l.logger.Debug("Error reading dashboard data", "error", err, "dashboardId", row.Id, "dashboardSlug", row.Slug)
```

Comment:
Should we log it on `info` instead, at least for a while? I think we won't be able to see debug logs on HG

Reply:
Yep 👍

Reply:
Logging on Warn now